src/firefly/js/ui/ValidationField.jsx

```diff
@@ -27,3 +28,4 @@ const propTypes= {
 };
 
 export const ValidationField= fieldGroupConnector(InputFieldView,getProps,propTypes);
+export const ValidationTextArea= fieldGroupConnector(InputAreaFieldView,getProps,propTypes);
```

Comment:
is 'InputAreaFieldConnected' (in file InputAreaField.jsx) the same as 'ValidationTextArea'?